README.adoc

```diff
@@ -39,3 +39,159 @@ Your branch should be prefixed with the version number, for example `4.3-new-exa
 When you are finished with your changes push the branch to your fork and create a Pull Request.
 Please add at least one approver.
 
+== Guidelines for writing Neo4j notifications
+
+When you want to add a new Notification code, you update the `neo4j` repo and this repo `docs-status-codes` by adding the notification details and examples.
+Each notification comprises the following components, some of which are documented only here in this repo.
+
+|===
+| Code| Neo.ClientNotification.Statement.CartesianProduct
+| Title|This query builds a cartesian product between disconnected patterns.
+| SeverityLevel| INFORMATION
+| Category| PERFORMANCE
+(For all categories and severity levels, see https://github.com/neo4j/docs-status-codes/blob/dev/modules/ROOT/pages/notifications/index.adoc)
+| Description |(Causes, consequences, a simple solution, or a reference to the Status Codes documentation)
+| Example| (Status Codes documentation only)
+| Suggestions for improvement| (Status Codes documentation only)
+|===
+
+The following are guidelines on how to write each of the notification components.
+
+=== General guidelines
+
+* Regardless of the notification category, each notification should include all the components.
+* Always use Present Simple tense and active voice if possible.
```

Comment:
Hm, not that I think about it - maybe we should add an example here like Louise suggested. I don't think everyone will understand this 😄 We could also add a link to some page describing it.